papaparse.js

```diff
@@ -969,7 +987,7 @@
 
 		function parseDynamic(field, value)
 		{
-			if ((_config.dynamicTyping[field] || _config.dynamicTyping) === true)
+			if (_config.dynamicTyping.isIncluded(field))
```

Comment:
As one of the objectives of Paparse is to be fast, I think we should cache it to prevent the user complaing about slow callbacks. Maybe it can be documented (see gh-pages branch)